incubator/bitcoind/templates/_helpers.tpl

```diff
@@ -0,0 +1,42 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "bitcoind.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "bitcoind.fullname" -}}
```

Comment:
Please compare with the `_helpers.tpl` file you get with `helm create` using Helm 2.8+ and update accordingly.

Reply:
Updated